### PR TITLE
Module exports

### LINKED ISSUE #13: Does not work in meteor.js
We are running into an issue when using video.js in meteorjs because of this library.

This package is failing to load because `root.VTTRegion` is undefined here:
 https://github.com/videojs/vtt.js/blob/master/lib/vttregion-extended.js#L29

If I add a 

```
module.exports.VTTRegion = this.VTTRegion;
```

at the very end of https://github.com/videojs/vtt.js/blob/master/lib/vttregion.js#L138 it all works nicely.

Would you be open to adding that modules.exports definition if I make a PR?

### PR DESCRIPTION
**Pull request by gkatsev**
Fixes #13.

### CODE REVIEW

**Comment by gkatsev** [comment]
A lot easier to review this with https://github.com/videojs/vtt.js/pull/14/files?w=1

File: lib/vtt.js [hunk 1]
```diff
@@ -16,1319 +16,1314 @@
...
+    for (var i = 0; i < cues.length; i++) {
+      cue = cues[i];
 
-      for (var i = 0; i < cues.length; i++) {
-        cue = cues[i];
+      // Compute the intial position and styles of the cue div.
+      styleBox = new CueStyleBox(window, cue, styleOptions);
+      paddedOverlay.appendChild(styleBox.div);
 
-        // Compute the intial position and styles of the cue di
```
Comment (gkatsev) [inline lib/vtt.js]: `WebVTT` gets set on the global object (really window) in `browser-index`.

File: lib/vttcue.js [hunk 2]
```diff
@@ -14,296 +14,292 @@
...
-  };
+VTTCue.prototype.getCueAsHTML = function() {
+  // Assume WebVTT.convertCueToDOMTree is on the global.
+  return WebVTT.convertCueToDOMTree(window, this.text);
+};
 
-  root.VTTCue = root.VTTCue || VTTCue;
-  vttjs.VTTCue = VTTCue;
-}(this, (this.vttjs || {})));
+module.exports = VTTCue;
```
Comment (gkatsev) [inline lib/vttcue.js:305]: VTTCue gets set correctly on `vttjs` and `window` inside of `browser-index`.

File: lib/vttregion.js [hunk 3]
```diff
@@ -14,125 +14,121 @@
...
-    });
-  }
+    }
+  });
+}
 
-  root.VTTRegion = root.VTTRegion || VTTRegion;
-  vttjs.VTTRegion = VTTRegion;
-}(this, (this.vttjs || {})));
+module.exports = VTTRegion;
```
Comment (gkatsev) [inline lib/vttregion.js:134]: `VTTRegion` gets set correctly inside of `browser-index`.

**Review by misteroneill — APPROVED** [review]

**Pull request merged**